sections/04_study.md

```diff
@@ -605,5 +605,65 @@ unlabeled (microbiome experiments) examples.
 
 ### Sequencing and variant calling
 
-*We have one nanopore paper in the issues and very recent work on variant calling
-that looks worthy of inclusion.*
+While we have so far discussed the role of deep learning in analyzing genomic
+data, deep learning approaches can also substantially improve our ability to
+obtain the genomic data itself. We will discuss two specific challenges: calling
+SNPs (single nucleotide polymorphisms) and indels (insertions and deletions)
+with high specificity and sensitivity, and improving the accuracy of new types
+of data such as nanopore sequencing. These two tasks are critical for studying
+rare variation, allele-specific transcription and translation, and splice site
+mutations, among others. In the clinical realm, sequencing of rare tumor clones
+and other genetic diseases will require accurate calling of SNP and indels.
+
+Current methods achieve relatively high (>99%) precision at 90% recall for SNPs
+and indel calls from Illumina short-read data, yet this leaves a large number of
```

Comment:
Yep I'll re-reference the Poplin paper, which benchmarked the previous state of the art algorithms (GATK).